pjmedia/include/pjmedia/sdp.h

```diff
@@ -489,12 +489,12 @@ typedef struct pjmedia_sdp_media pjmedia_sdp_media;
  *
  * @param media     The media description.
  * @param buf       The buffer.
- * @param size      The buffer length.
+ * @param legth     The buffer length.
```

Comment:
typo: `legth`

Reply:
I'm not sure why you need to change the parameter name here.

IMO, it's better to leave existing API's parameter names untouched, unless there's a reason for the change.